alphagov/paas-rubbernecker
# [BUG] Fix card dealing and counters

## What

Currently, the application would not distribute or update the cards
correctly.

We're also adding extra counters for columns without them in order to
easily identify the amount of stories in each state.

## How to review

- Build `make build` and run `./bin/rubbernecker`
- Visit `localhost:8080`
- Play with story state
- See the application updating counters and cards correctly
